ocean/nemo/nemolite2d/manual_versions/psykal_ocl/fortran/gocean2d_io_mod.f90

```diff
@@ -0,0 +1 @@
+../../../psykal/gocean2d_io_mod.f90
```

Comment:
I think the Makefile should have a set of shared files from a different directory, rather than sym linking them.

Reply:
Yes. I've re-worked things so that we now have a `nemolite2d/common/` directory that contains these files.